clusterman/cli/util.py

```diff
@@ -0,0 +1,23 @@
+import argparse
+import signal
+
+import colorlog
+
+logger = colorlog.getLogger(__name__)
+
+
+def timeout_wrapper(main):
```

Comment:
nit: The naming specifically assumes a main function that accepts an args namespace. I think you can just make the `timeout_wrapper` accept any function, and `wrapper` accept any and all arguments. It'll just pass those to the function.

Reply:
I kinda disagree here... I think this function should be explicit that it's wrapping the `main` function of a CLI subcommand.  If it changes to be more generally-used in the future, we can update the terminology then.